### PR TITLE
[CSYS-450] Incorporating the Fluent UI Icons

### PR DESCRIPTION
**Pull request by tiidadavena**
**How to test:** [CSYS-450](https://labcodes.atlassian.net/browse/CSYS-450)

**Description of your solution:** Inserting the [Fluent UI](https://developer.microsoft.com/en-us/fluentui) iconography into the Confetti Icon's system. 

**Screenshots (when applicable):** I recommend to run the project locally to view all the icons' list: 
image

[CSYS-450]: https://labcodes.atlassian.net/browse/CSYS-450?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/confetti-ds/scss/components/_lab-badge.scss [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
   -webkit-mask-image: url($badges-svg);
   mask-image: url($badges-svg);
   -webkit-mask-repeat: no-repeat;
   mask-repeat: no-repeat;
-  height: $icon-size;
-  width: $icon-size;
-  font-size: $icon-size;
+  //height: $icon-size;
+  //width: $icon-size;
+  //font-size: $icon-size;
```
Comment (lucianoratamero) [inline packages/confetti-ds/scss/components/_lab-badge.scss:12]: I know there is a PR removing the badges, but, as a general advice, it's always best to not commit commented code.
we can always go back to older commits to get the code back, if necessary.

File: packages/confetti-ds/src/Button/__snapshots__/OutlineButton.test.tsx.snap [hunk 2]
```diff
@@ -39,9 +39,6 @@ exports[`OutlineButton renders as expected when passing a magnifying-glass icon
   onClick={[Function]}
   type="button"
 >
-  <span
-    className="lab-icon lab-icon--magnifying-glass lab-icon--white lab-icon--petit lab-btn__icon"
-  />
   Test Outline magnifying-glass Button
```
Comment (lucianoratamero) [inline packages/confetti-ds/src/Button/__snapshots__/OutlineButton.test.tsx.snap:42]: something seems weird here. why did the icon stop rendering? is this correct?
maybe you forgot to change the tests for this component.
could you double check this?

File: packages/confetti-ds/src/Tags/SimpleTag.test.tsx [hunk 3]
```diff
@@ -48,24 +48,18 @@ describe("SimpleTag", () => {
     expect(wrapper).toContain("lab-tag--vivid");
   });
 
-  it("renders as expected with a magnifying-glass icon", async () => {
+  it("renders as expected with a lupe icon", async () => {
```
Comment (lucianoratamero) [inline packages/confetti-ds/src/Tags/SimpleTag.test.tsx:51]: I'm pretty sure lupe is not an english word :B
https://en.wiktionary.org/wiki/lupe#English

**Review by lucianoratamero — CHANGES_REQUESTED** [review]
is the documentation supposed to be here?
I'm pretty sure we should add something about using fluent UI's icons on our readme.

File: packages/confetti-ds/src/Tags/SimpleTag.test.tsx (hunk 3 above)
Comment (tiidadavena) [reply packages/confetti-ds/src/Tags/SimpleTag.test.tsx:51]: Sorry for the mess! Searching around here, the previous "magnifying-glass" should be replaced by the equivalent "Search" icon.

File: packages/confetti-ds/src/Button/__snapshots__/OutlineButton.test.tsx.snap (hunk 2 above)
Comment (tiidadavena) [reply packages/confetti-ds/src/Button/__snapshots__/OutlineButton.test.tsx.snap]: snapshot was updated 🥳

**Pull request merged**